nixos/modules/services/monitoring/gatus.nix

```
@@ -0,0 +1,147 @@
+{ pkgs, lib, config, ... }:
+let
+  cfg = config.services.gatus;
+
+  settingsFormat = pkgs.formats.yaml { };
+
+  inherit (lib)
+    getExe
+    literalExpression
+    mdDoc
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOptionMD
+    types
+    ;
+in
+{
+  options = {
+    services.gatus = {
```

you can remove a level of indent here
```suggestion
  options.services.gatus = {
```